book/http.md

```diff
@@ -736,7 +737,7 @@ Encrypted connections
 =====================
 
 So far, our browser supports the `http` scheme. That's a pretty common
-scheme. But more and more, websites are migrating to the
+scheme. But more and more websites are migrating to the
```

Comment:
Curiously, both "But, more and more," and also "But more and more" are grammatically correct, mean different things, and also those things are both true. I couldn't decide which one we really meant and went with this one, which is what the copyeditor suggested.